packages/eslint-plugin/src/rules/class-literal-property-style.ts

```diff
@@ -110,21 +97,30 @@ export default util.createRule<Options, MessageIds>({
           context.report({
             node: node.key,
             messageId: 'preferGetterStyle',
-            fix(fixer) {
-              const sourceCode = context.getSourceCode();
-              const name = sourceCode.getText(node.key);
-
-              let text = '';
-
-              text += printNodeModifiers(node, 'get');
-              text += node.computed ? `[${name}]` : name;
-              text += `() { return ${sourceCode.getText(value)}; }`;
-
-              return fixer.replaceText(node, text);
-            },
           });
         },
       }),
     };
   },
 });
+
+function getPropertyName(key: TSESTree.PropertyName): unknown {
+  switch (key.type) {
+    case AST_NODE_TYPES.PrivateIdentifier:
+    case AST_NODE_TYPES.Identifier:
+      return key.name;
+    case AST_NODE_TYPES.Literal:
+      return key.raw;
+    default:
+      return undefined;
```

Comment:
🤔 this is never hit by tests. But the TypeScript types believe it can be hit. One of the two following cases are probably true:

* The types are wrong, and should be improved so that this `default: return undefined;` can be removed
* The unit tests are missing some cases that should be added